stream/src/placeholder/instance.rs

```diff
@@ -150,20 +150,14 @@ impl<C: Scheme, Si: Sink, St: Stream> Instance<C, Si, St> {
             Sender {
                 cipher: self.cipher.clone(),
                 sink: self.sink,
-
                 max_message_size: self.config.max_message_size,
-                dialer: self.dialer,
-                iter: 0,
-                seq: 0,
+                nonce: nonce::Info::new(self.dialer),
             },
             Receiver {
                 cipher: self.cipher,
                 stream: self.stream,
-
                 max_message_size: self.config.max_message_size,
-                dialer: self.dialer,
-                iter: 0,
-                seq: 0,
+                nonce: nonce::Info::new(!self.dialer),
```

Comment:
This change is not a difference in behavior (as we performed `!self.dialer` inside of `peer_nonce`). Calling it out because it "looks like".